scipy/special/add_newdocs.py

```diff
@@ -876,12 +876,61 @@ def add_newdoc(name, doc):
     """)
 
 add_newdoc("betaincinv",
-    """
-    betaincinv(a, b, y)
+    r"""
+    betaincinv(a, b, y, out=None)
 
-    Inverse function to beta integral.
+    Inverse of the incomplete beta function.
 
-    Compute `x` such that betainc(a, b, x) = y.
+    Computes :math:`x` such that:
+    
+    .. math::
+    
+        y = \frac{\Gamma(a+b)}{\Gamma(a)\Gamma(b)} \int_0^x 
+        t^{a-1}(1-t)^{b-1}dt,
```

Comment:
Would it be better here to write `y = I_x(a, b) = \frac{  ...` and say `I_x` is the normalized incomplete beta function?